webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/detail/AnnotationDetailEditorPanel.java

```diff
@@ -776,14 +773,16 @@ public JCas getEditorCas()
         if (state.getMode().equals(Mode.ANNOTATION) || state.getMode().equals(Mode.AUTOMATION)
                 || state.getMode().equals(Mode.CORRECTION)) {
 
+            // FIXME de.tudarmstadt.ukp.clarin.webanno.api.DocumentService readAnnotationCas is
```

Comment:
Do we need a comment for that? The IDE should already be warning about deprecation.